### PR TITLE
Bump version to v5.3.3

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**

### CODE REVIEW

File: rpm/spec.in [hunk 1]
```diff
@@ -84,6 +84,9 @@ mkdir -p $RPM_BUILD_ROOT/
 %{_includedir}/libdrizzle-5.1/visibility.h
 
 %changelog
+* Fri Dec 02 2016 Andreas Bok Andersen <andreas.bok@sociomantic.com - 5.3.3
```
Comment (ben-palmer-sociomantic) [inline rpm/spec.in]: Are you missing a last `>` on your email?

**Comment by ben-palmer-sociomantic** [comment]
One comment, otherwise LGTM.

File: rpm/spec.in (hunk 1 above)
Comment (andreas-bok-sociomantic) [reply rpm/spec.in]: Indeed, also missing for my other changelogs entries :-1:

**Comment by andreas-bok-sociomantic** [comment]
Fixed missing `>`

**Pull request merged**